pkg/buses/cache.go

```diff
@@ -96,3 +96,19 @@ func (c *Cache) RemoveSubscription(subscription *channelsv1alpha1.Subscription)
 	subscriptionRef := NewSubscriptionReference(subscription)
 	delete(c.subscriptions, subscriptionRef)
 }
+
+func (c *Cache) AllChannels() []*channelsv1alpha1.Channel {
```

Comment:
the `cache.go` file currently has good test coverage, can we add a couple tests?